spec/system/feedback_spec.rb

```diff
@@ -0,0 +1,74 @@
+require "rails_helper"
+
+RSpec.describe "Feedback component" do
+  let(:assessment_code) { :assessment_code }
+
+  before do
+    driven_by(:headless_chrome)
+  end
+
+  describe "satisafaction feedback" do
+    before do
+      stub_request(:post, %r{v6/assessments\z}).to_return(
+        body: FactoryBot.build(:api_result, eligible: "eligible").to_json,
+        headers: { "Content-Type" => "application/json" },
+      )
+    end
+
+    context "when on the results page" do
+      it "I can successfully submit satisfaction feedback" do
+        start_assessment
+        fill_in_forms_until(:check_answers)
+        click_on "Submit"
+        expect(page).to have_content("Were you satisfied with this service?")
+        click_on "Yes"
+        expect(page).to have_content("Thank you for your feedback")
```

Comment:
It might be a good idea to verify that a record has been saved in our database here. Either by checking for SatisfactionFeedback objects or by in the test, logging in to the admin back end and verifying that they appear on screen.

Reply:
to chat about when pairing